pkg/autoscaler/autoscaler_server.go

```diff
@@ -157,6 +178,7 @@ func (s *AutoScaler) readConfigFileIfChanged() ([]byte, error) {
 	if os.SameFile(fi, s.lastFileInfo) {
 		return nil, nil
 	}
+	s.lastFileInfo = fi
```

Comment:
Does this mean the config update logic was broken before this? :)

Reply:
No, it just unnecessary re-parsed the file and produced unnecessary logs.